presentation/common/src/presentation-common/Diagnostics.ts

```diff
@@ -21,8 +21,11 @@ export interface ClientDiagnostics extends Diagnostics {
 
 /** @alpha */
 export interface DiagnosticsOptions {
-  /** Measure performance */
-  perf?: boolean;
+  /**
+   * Flag specifying that performance should be measured, or
+   * minimum duration in milliseconds for which performance metric should be included.
+   */
+  perf?: boolean | { duration: number };
```

Comment:
Similar to the addon, I think `duration` is not the right name for the attribute. I suggest "minimum duration" or "duration threshold"